dnf-plugins-core.spec

```diff
@@ -1,6 +1,6 @@
 %{?!dnf_lowest_compatible: %global dnf_lowest_compatible 1.1.4}
 %{?!dnf_not_compatible: %global dnf_not_compatible 2.0}
-%global hawkey_version 0.6.1
+%global hawkey_version 0.7.0
```

Comment:
@jsilhan You mean `/etc/distro.repos.d`, right?

As for the rest of this list, I agree with it. What's the criteria required to get libhif 0.7.0 in a state to be released?